### PR TITLE
Mentor

### PR DESCRIPTION
**Pull request by jnlaguerre23**

### CODE REVIEW

**Comment by jnlaguerre23** [comment]
@levonlloyd  Please review before I push onto master and let me know if you agree with the changes for Mentor Model & form

File: app/controllers/mentor_controller.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  def new
+      @mentor=Mentor.new
+  end
+
+  def create
+    @mentor=Mentor.new
+    @user = User.find(session[:user_id])
+    @mentor.user=@user
+    @mentor.email=@user.email
```
Comment (levonlloyd) [inline app/controllers/mentor_controller.rb:12]: this is unnecessary.  If you need the email of a mentor elsewhere, you can do mentor.user.email

File: db/migrate/20150428031431_add_user_id_to_mentor.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddUserIdToMentor < ActiveRecord::Migration
```
Comment (levonlloyd) [inline db/migrate/20150428031431_add_user_id_to_mentor.rb:1]: There should only be one migration per PR.  please add this column to the above migration

**Comment by levonlloyd** [comment]
@jnlaguerre23 just a couple of comments, otherwise, it looks good.

**Pull request merged**